iogt/templates/core/tags/section_listing_homepage.html

```diff
@@ -1,19 +1,64 @@
-{% load wagtailcore_tags static i18n %}
+{% load wagtailcore_tags wagtailimages_tags comments static core_tags i18n %}
 
 {% if sections %}
-<div id="section_menu" class="section-menu">
-  <div class="section-heading">
-    <a href="#"><img src="{% static 'img/iogt_logo_2.png' %}" class="logo" alt="The Internet of Good Things" width="40%" /></a>
-  </div>
-  <ul class="menu-list">
-    {% for section in sections %}
-    <li class="menu-list__item menu-list__item{{section.get_effective_extra_style_hints}}">
-      <a href="{% pageurl section %}">
-        {{section.title}}
-        <span class="menu-list__desc">{{section.description}}</span>
-      </a>
-    </li>
-    {% endfor %}
-  </ul>
-</div>
+<h3 class="heading heading--homepage">{% trans "Recommended" %}</h3>
+<div class="section-list section-list__homepage">
```

Comment:
This should be a modifier on the original, Eg: `section-list--homepage`

We have already created a convention for naming these; it's usually `article-list` since we can re-use that in several places: 

1. in latest articles
2. in article list homepage
3. in search
4. in section list page.

My recommendation for differentiating them would be to add modifers, but to keep them as `article-list`